src/tools/web/elektrad/router.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"net/http/pprof"
+
+	"github.com/gorilla/mux"
+)
+
+func setupRouter(app *server) http.Handler {
+	r := mux.NewRouter()
+	AttachProfiler(r)
+
+	r.Use(handleMiddleware(app.pool))
+
+	r.HandleFunc("/version", app.getVersionHandler).Methods("GET")
+
+	r.HandleFunc("/kdb", app.getKdbHandler).Methods("GET")
+	r.HandleFunc("/kdb/{path:.*}", app.getKdbHandler).Methods("GET")
+	r.HandleFunc("/kdb/{path:.*}", app.putKdbHandler).Methods("PUT")
+	r.HandleFunc("/kdb/{path:.*}", app.deleteKdbHandler).Methods("DELETE")
+
+	r.HandleFunc("/kdbFind/{path:.*}", app.getFindHandler).Methods("GET")
+
+	r.HandleFunc("/kdbMv/{path:.*}", app.postMoveHandler).Methods("POST")
+
+	r.HandleFunc("/kdbMeta/{path:.*}", app.postMetaHandler).Methods("POST")
+	r.HandleFunc("/kdbMeta/{path:.*}", app.deleteMetaHandler).Methods("DELETE")
+
+	return r
+}
+
+func AttachProfiler(router *mux.Router) {
```

Comment:
Is this something that should be turned off for production/deployment (due to e.g. security?)

Reply:
good catch - removed